components/helm-broker/internal/bundle/automock/broker_syncer.go

```diff
@@ -3,18 +3,18 @@ package automock
 
```

Comment:
Should it be removed? similar file exist here `internal/controller/automock/broker_syncer.go`